azure.gemspec

```diff
@@ -50,3 +49,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency('yard',                '~> 0.8')
   s.add_development_dependency('vcr',                 '~> 3.0')
 end
+
```

Comment:
how would we do that? at present, the only way is to use rubymine.  i am still waiting for mine.